File: app/src/main/java/com/pluu/webtoon/di/RealmHelperModule.java
package com.pluu.webtoon.di;

import com.pluu.webtoon.db.RealmHelper;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

/**
 * Realm Helper Module
 * Created by PLUUSYSTEM-NEW on 2016-12-04.
 */
@Module
public class RealmHelperModule {

    @Provides
    @Singleton
    static RealmHelper provideRealmHelper() {
       return new RealmHelper();
    }

}

File: app/src/main/java/com/pluu/webtoon/di/RealmHelperModule.kt
package com.pluu.webtoon.di

import com.pluu.webtoon.db.RealmHelper
import dagger.Module
import dagger.Provides
import javax.inject.Singleton

/**
 * Realm Helper Module
 * Created by pluu on 2017-04-30.
 */
@Module
class RealmHelperModule {

    @Provides
    @Singleton
    fun provideRealmHelper(): RealmHelper = RealmHelper()

}
